Detections/SecurityEvent/ExcessiveLogonFailures.yaml

```diff
@@ -66,6 +67,72 @@ query: |
   | order by sum_CountToday desc nulls last 
   | extend timestamp = StartTime, AccountCustomEntity = Account, HostCustomEntity = WorkstationName
+  ),
+  (WindowsEvent
+  | where TimeGenerated >= ago(endtime)
+  | where EventID == 4625 and  not(EventData has_any ("127.0.0.1", "::1"))
+  | extend TargetUserSid = tostring(EventData.TargetUserSid)
+  | extend AccountType=case(EventData.TargetUserName endswith "$" or TargetUserSid in ("S-1-5-18", "S-1-5-19", "S-1-5-20"), "Machine", isempty(TargetUserSid), "", "User")
+  | where AccountType =~ "User"
+  | extend IpAddress = tostring(EventData.IpAddress)
+  | where IpAddress !in ("127.0.0.1", "::1")
+  | extend Account = strcat(tostring(EventData.TargetDomainName),"\\", tostring(EventData.TargetUserName))
+  | extend LogonType = toint(EventData.LogonType)
+  | extend LogonTypeName=case(LogonType==2,"2 - Interactive", LogonType==3,"3 - Network", LogonType==4, "4 - Batch",LogonType==5, "5 - Service", LogonType==7, "7 - Unlock", LogonType==8, "8 - NetworkCleartext", LogonType==9, "9 - NewCredentials", LogonType==10, "10 - RemoteInteractive", LogonType==11, "11 - CachedInteractive",tostring(LogonType))
+  | extend SubStatus = tostring(EventData.SubStatus)
+  | extend WorkstationName = tostring(EventData.WorkstationName)
+  | extend NewProcessName = tostring(EventData.NewProcessName)
+  | extend Process=tostring(split(NewProcessName, '\\')[-1])
+  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), CountToday = count() by EventID, Account, LogonTypeName, SubStatus, AccountType, Computer, WorkstationName, IpAddress, Process
+  | join kind=leftouter (
+      WindowsEvent 
+      | where TimeGenerated between (ago(starttime) .. ago(endtime))
+      | where EventID == 4625 and not(EventData has_any ("127.0.0.1", "::1"))
+      | extend TargetUserSid = tostring(EventData.TargetUserSid)
+      | extend AccountType=case(EventData.TargetUserName endswith "$" or TargetUserSid in ("S-1-5-18", "S-1-5-19", "S-1-5-20"), "Machine", isempty(TargetUserSid), "", "User")
+      | where AccountType =~ "User"
+      | extend IpAddress = tostring(EventData.IpAddress)
+      | where IpAddress !in ("127.0.0.1", "::1")
+      | extend Account = strcat(tostring(EventData.TargetDomainName),"\\", tostring(EventData.TargetUserName))
+      | extend LogonType = toint(EventData.LogonType)
+      | extend LogonTypeName=case(LogonType==2,"2 - Interactive", LogonType==3,"3 - Network", LogonType==4, "4 - Batch",LogonType==5, "5 - Service", LogonType==7, "7 - Unlock", LogonType==8, "8 - NetworkCleartext", LogonType==9, "9 - NewCredentials", LogonType==10, "10 - RemoteInteractive", LogonType==11, "11 - CachedInteractive",tostring(LogonType))
+      | extend SubStatus = tostring(EventData.SubStatus)
+      | extend WorkstationName = tostring(EventData.WorkstationName)
+      | extend NewProcessName = tostring(EventData.NewProcessName)
+      | extend Process=tostring(split(NewProcessName, '\\')[-1])
```

Comment:
You actually don't need process here, which might be a bug in the original query